pkg/report/linux.go

```diff
@@ -1322,7 +1322,14 @@ var linuxOopses = append([]*oops{
 						compile("(Local variable .* created at:|Uninit was created at:)"),
 						parseStackTrace,
 					},
-					skip: []string{"alloc_skb"},
+					// This is a bit heavy-handed, but in practice KMSAN origin
```

Comment:
There is also "KMSAN origin in" alt title in the previous case "KMSAN: kernel-usb-infoleak".
I think we need to do the same there. Otherwise the same origin stack will be handled differently for usb-infoleak and all other KMSAN reports. As the result bugs with the same origin won't be merged together.